frontend/src/Board.css

```diff
@@ -0,0 +1,82 @@
+.board {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  background: #2c2c2c;
+}
+
+.board__header {
+  display: flex;
+  width: 320px;
+  padding: 10px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #e3e3e3;
+  color: #2c2c2c;
+}
+
+.board__header__title {
+  display: flex;
+  width: 100%;
+  position: relative;
+  align-items: center;
+}
+
+.board__header__title__image {
+  display: inline-block;
+  height: 60px;
+  width: 60px;
+  margin-right: 5px;
+  background-color: #bbb;
+  border-radius: 50%;
+}
+
+.board__header__title__menu {
+  display: flex;
+  position: absolute;
+  justify-content: center;
+  align-items: center;
+  top: 0;
+  right: 0;
+  cursor: pointer;
+}
+
+.board__header__title__menu:hover * {
+  background-color: #2c2c2c;
+}
+
+.board__header__author {
+  display: flex;
+  width: 100%;
+  justify-content: flex-end;
+  align-items: center;
+  color: #6b6b6b;
+}
+
+.board__header__author__avatar {
+  display: inline-block;
+  height: 25px;
+  width: 25px;
+  margin-left: 5px;
+  background-color: #bbb;
+  border-radius: 50%;
+}
+
+.board__header__author__name:hover {
+  text-decoration: underline;
+  text-decoration-thickness: 1px;
+  cursor: pointer;
+}
+
+/* kropka potrzebna do stworzenia przycisku do rozwijania menu */
+.menu__dot {
+  display: inline-block;
+  height: 5px;
+  width: 5px;
+  margin-left: 2px;
+  background-color: #bbb;
+  border-radius: 50%;
+  transition: background-color 0.15s ease-out;
+}
```

Comment:
Cześć, ustaliliśmy, że stylowanie piszemy za pomocą styled components ;)
Zapoznaj się, proszę, bo mocno to się różni.
Dodatkowo taka uwaga, że jakbyś tworzył klasy w BEM, to nie możemy zagnieżdżać więcej niż jeden raz.
https://styled-components.com/

Reply:
O BEM to już rozmawialiśmy wielokrotnie i używałeś przecież w wielu komponentach;) Tu jest błędnie.

A tak w ogóle to używając styledComponents będziemy mogli zrezygnować z BEM ;)